main.py

```diff
@@ -1,25 +1,27 @@
 ### Assignment: Regex and Files
-##Name:
+##Name: K.King
 ##Date: 
 
 #don't forget to import regex
-
+import re
 
 ##(/5) Task 1: MODIFY the code below.
 #A# Change the regex from .* to what is required to capture an email address
 #B# Add a condition so that no blanks are printed.
 #C# Count the number of email addresses found and print a final output line.
 
+count = 0 
 def reEmail(fname):
-  
+  global count
   fhand = open(fname,'r')
   for line in fhand:
-    extr = re.findall('.*',line)
+    extr = re.findall(".*@\S\S\S\S\S\S\S\S\S",line)
```

Comment:
Your regex will extract "@apaulsTeacher" where there are 0 characters before the @.
`extr = re.findall('\S+@\S+',line)`